src/bastion/azext_bastion/custom.py

```diff
@@ -279,6 +280,9 @@ def rdp_bastion_host(cmd, target_resource_id, target_ip_address, resource_group_
             }
             response = requests.get(web_address, headers=headers)
             if not response.ok:
+                errorMessage = json.loads(response.content)['message']
```

Comment:
The "message" key may not exist in response.
```suggestion
                errorMessage = json.loads(response.content).get('message', None)
```